provisioning/roles/firefox/vars/main.yml

```diff
@@ -0,0 +1,16 @@
+firefox_dependencies:
+  - libgtk-3-0
+  - libdbus-glib-1-2
+
+# default version
+firefox_version: latest
```

Comment:
These should be in `defaults/main.yml` (otherwise you won't be able to change them in `parameters.yml`).